net/net.sh

```diff
@@ -149,6 +154,9 @@ build() {
     $MAYBE_DOCKER bash -c "
       set -ex
       scripts/cargo-install-all.sh farf \"$cargoFeatures\"
+      if [[ -n \"$customPrograms\" ]]; then
```

Comment:
nit: I think you can just do this: `if [[ -n $customPrograms ]]; then`, unlike `[` there's no need for addition escaping with `[[`